ASIM/schemas/entities/ASimApp.yaml

```diff
@@ -18,6 +18,12 @@ Fields:
   Follows: <<Role>>AppName
   Description: The type of the application.
 
+- Name: <<Role>>OriginalAppType
+  Type: string
+  Class: Optional
```

Comment:
I don't see we have Role defined for other fields  such as AppName, AppType in the same Entity, I feel OriginalAppType tied with AppName and App Type then why << Role >> is relevant only for this. Can you please clarify?

Reply:
@anki-narravula 
yes, removed it as other related fields are not having it.